test/cli_run_volume_test.go

```diff
@@ -422,3 +423,44 @@ func (suite *PouchRunVolumeSuite) TestRunCopyDataWithDR(c *check.C) {
 		c.Fatal("no copy image data, miss mail directory")
 	}
 }
+
+func (suite *PouchRunVolumeSuite) TestRunVolumesFromWithDR(c *check.C) {
```

Comment:
xxFromWithxx, maybe, there is a better naming for it.

Reply:
VolumesFrom is a funtion, you can see `pouch run -h`, it contains `--volumes-from`. 😆

Reply:
Well, it just has a little weird ^_^